.github/workflows/release.yaml

```diff
@@ -10,6 +10,7 @@ on:
   push:
     branches:
       - 14-22.04
+  workflow_dispatch:
```

Comment:
We need to be able to dispatch manually, to get updates from the snaps that don't really require changes on the rock's end.